src/sedtrails/configuration_interface/configuration_controller.py

```diff
@@ -16,10 +19,9 @@ class Controller(ABC):
 
     # TODO: Something to keep in mind in this class is that
     # if we can and it is convenient to have separted methods to retrieve parts of the configuration
-    # that go to the particle tracer and the transport converter. For example, if it is convenient to 
+    # that go to the particle tracer and the transport converter. For example, if it is convenient to
     # have a method that returns the configuration values that are only relevant for the particle tracer.
```

Comment:
we can remove this comments. We agreed to pass all configurations to other components.